src/components/Calculator.js

```diff
@@ -0,0 +1,28 @@
+const Calculator = () => (
+  <div className="calculator">
+    <div className="screen-output">0</div>
+    <div className="inputs-section">
+      <button className="btn-ac" type="button">AC</button>
+      <button className="btn-plus-les" type="button">+/-</button>
+      <button className="btn-percentage" type="button">%</button>
+      <button className="divide orange" type="button">/</button>
+      <button className="multiply orange" type="button">X</button>
+      <button className="less orange" type="button">-</button>
+      <button className="plus orange" type="button">+</button>
+      <button className="equal orange" type="button">=</button>
+      <button className="nine" type="button">9</button>
+      <button className="eigth" type="button">8</button>
+      <button className="seven" type="button">7</button>
+      <button className="six" type="button">6</button>
+      <button className="five" type="button">5</button>
+      <button className="four" type="button">4</button>
+      <button className="three" type="button">3</button>
+      <button className="two" type="button">2</button>
+      <button className="one" type="button">1</button>
+      <button className="zero" type="button">0</button>
+      <button className="point" type="button">.</button>
+    </div>
+  </div>
+);
```

Comment:
- [ ] It would be great if Calculator.js should have at least 1 child component to make your code reusable and make you understand how a react app deals with the components and it is required in the project requirements 👍

Reply:
Thanks, I will turn Input-section then into a function component and then I will nest it inside the Calculator, Thanks for your advice! I do appreciate it!